modules/sbo-categories-of-exposable-binding-data.adoc

```diff
@@ -176,9 +193,9 @@ The following example shows how the previous entries of a collection in annotati
 /bindings/<binding-name>/uri_404 => black-hole.example.com
 ----
 
-Following is the example for exposing the entries of a collection with the key and value from each entry as OLM descriptors:
+Following is the example for exposing the entries of a collection with the key and value from each entry through OLM descriptors:
```

Comment:
```suggestion
The following example shows how to expost the entries of a collection with the key and value from each entry through OLM descriptors:
```
Rephrase to be consistent with phrasing used in prior examples.

Reply:
There is a typo in the suggestion `expost`, should be `expose`.

Reply:
Thanks a lot, @pmacik, and @bburt-rh!